core/core.go

```diff
@@ -296,7 +297,22 @@ func getBazelVersion() (string, error) {
 		}
 	}
 
-	return "latest", nil
+	fallbackVersion := os.Getenv("USE_BAZEL_FALLBACK_VERSION")
+	if len(fallbackVersion) == 0 {
+		fallbackVersion = "silent:latest"
+	}
+	if strings.HasPrefix(fallbackVersion, "error:") {
```

Comment:
it would be easier to split fallbackVersion on the colon and then (if there are two parts) look at part 0 and 1